aws-s3-transfer-manager/src/error.rs

```diff
@@ -44,6 +44,11 @@ pub enum ErrorKind {
 
     /// child operation failed (e.g. download of a single object as part of downloading all objects from a bucket)
     ChildOperationFailed,
+
+    /// The operation is being canceled because the user explicitly called `.abort` on the handle,
+    /// or a child operation failed with the abort policy. The operation is now transitioning into
```

Comment:
this wording ("the operation is now transition into...") implies the user may still need to do something.

Reply:
Ah got it, removed the sentence in a6fbdcf